README.rst

```diff
@@ -42,12 +42,12 @@ The built documentation will be available at::
 
 Any good internet browser will be able to display the documentation.
 
-To build the pdf form of the documentation simply execute the following command::
+To build the PDF form of the documentation simply execute the following command::
 
   make latexpdf
 
-Note: Creating the pdf form of the documentation requires that latexmk is available, information on latexmk can be found at https://ctan.org/pkg/latexmk/.
+Note: Creating the PDF form of the documentation requires that latexmk is available, information on latexmk can be found at https://ctan.org/pkg/latexmk/.
 
 Note: The documentation built with `make html` is different to the documentation built with `make latexpdf`.
-The html form of the documentation is the full form being both the normative specification and the informative specification.
-Whilst the pdf form of the documentation is solely the normative specification.
+The HTML form of the documentation is the full form being both the normative specification and the informative specification.
+Whilst the PDF form of the documentation is solely the normative specification.
```

Comment:
Sorry, ignore earlier comment on earlier commit re capitals of HTML and PDF ... but the second part still stands.  These two sentences need to be one (with a ; or , between) or remove the "Whilst" and start with "The".